src/lib/handle-result.ts

```diff
@@ -92,11 +106,30 @@ export function createSection(entry?: TestNameEntry): SectionSummary {
 }
 
 export function createUpdateFunctionSummary(
+    functionLeaders: FunctionLeaders,
     functionMap: FunctionMap,
     functions: FunctionEntry[],
-    entry: FunctionLocationEntry | FunctionExecutionEntry
+    entry: FunctionAliasEntry | FunctionLocationEntry | FunctionExecutionEntry
 ): void {
     const functionSummary = functionMap.get(entry.name)
+    if (entry.variant === Variant.FunctionAlias) {
+        // biome-ignore lint/style/noNonNullAssertion: Leader always comes before aliases of a function
+        const functionLeader = functionLeaders.get(entry.index)!
```

Comment:
This may be technically correct, but it would be better to make sure it doesn't crash just because some generator didn't use the expected order.

It would be great if the new implementation would fallback gracefully like the current one does for `FN` fields.